dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -245,6 +248,32 @@ public String toString() {
         };
     }
 
+    @Override
+    public StickyChannels getStickyChannels2(String serviceName) {
+        Channel channel = getChannel(serviceName);
+        Supplier<Supplier<Channel>> endpointChannelFactorySupplier = StickyEndpointChannels2.create(
+                endpoint -> params.runtime().clients().bind(channel, endpoint));
+
+        return new StickyChannels() {
+            @Override
+            public StickyChannelFactory2 create() {
+                Supplier<Channel> channelSupplier = endpointChannelFactorySupplier.get();
+                return new StickyChannelFactory2() {
+                    @Override
+                    public <T> T create(Class<T> clientInterface) {
+                        return Reflection.callStaticFactoryMethod(
+                                clientInterface, channelSupplier.get(), params.runtime());
+                    }
+                };
+            }
+
+            @Override
+            public String toString() {
+                return "StickyChannels{}";
```

Comment:
I'd either add the relevant data from the delegate channel/factory, or remove the toString.